src/overlays/screen.rs

```diff
@@ -110,8 +112,12 @@ impl InteractionHandler for ScreenInteractionHandler {
 
         app.hid_provider.send_button(btn, pressed);
 
-        let pos = self.mouse_transform.transform_point2(hit.uv);
-        app.hid_provider.mouse_move(pos);
+        if !app.session.config.focus_follows_mouse_mode
+            || app.input_state.pointers[hit.pointer].now.move_mouse {
+            let pos = self.mouse_transform.transform_point2(hit.uv);
+            app.hid_provider.mouse_move(pos);
+        }
+
```

Comment:
this is for clicks, so i'm not sure if we need the check there. i added this so that there's no discrepancy of cursor position when clicking